s3/init_test.go

```diff
@@ -12,15 +15,44 @@ const (
 
 var testStore *Storage
 
+const (
+	// minioImage is the default image used for running S3 in tests.
+	minioImage              = "docker.io/minio/minio:RELEASE.2024-08-17T01-24-54Z"
```

Comment:
Use latest

Reply:
Just copied the version from the current minio module. We could keep it as is, and bump the versions all together in a follow-up